Jenkinsfile

```diff
@@ -35,12 +35,27 @@ pipeline {
 
     stage('Scan Secretless') {
       parallel {
-        stage('Scan Secretless Image') {
+        stage('Scan Secretless Images') {
           steps {
             scanAndReport("secretless-broker:latest", "HIGH")
           }
         }
 
+        stage('Scan Secretless Quickstart') {
+          steps {
+            scanAndReport("secretless-broker-quickstart:latest", "CRITICAL")
+          }
+        }
+
+        stage('Scan Secretless RedHat') {
+          steps {
+            script {
+              TAG = sh(returnStdout: true, script: '. bin/build_utils && full_version_tag')
+            }
+            scanAndReport("secretless-broker-redhat:${TAG}", "NONE")
+          }
+        }
```

Comment:
is there any reason not to run these scans in parallel? if not, I have a strong preference for parallelizing this stage.

Reply:
They're already in parallel - the original Jenkinsfile parallelized the first trivy scan and the Go sec run.